docs/services/slam/cartographer/_index.md

```diff
@@ -85,19 +88,83 @@ Paste the following into the **Attributes** field of your new service:
 }
 ```
 
+{{% /tab %}}
+
+{{% tab name="MacOS" %}}
+
+```json
+{
+  "data_dir": "/Users/<YOUR_USERNAME>/<CARTOGRAPHER_DIR>",
+  "delete_processed_data": true,
+  "use_live_data": true,
+  "sensors": ["rplidar"],
+  "config_params": {
+    "mode": "2d"
+  }
+}
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+Select the **Raw JSON** mode and copy/paste the following `"modules"` JSON to your existing SLAM and RPLIDAR configuration:
+
+{{< tabs name="Add Cartographer Service Live Module">}}
+{{% tab name="Linux/MacOS x86_64" %}}
+
+```json
+"modules": [
+  // { ...}, YOUR RPLIDAR MODULE,
+  {
+    "executable_path": "/usr/local/bin/cartographer-module",
```

Comment:
Just confirming that the path is /usr/local/bin on MacOS x86_64?

Reply:
It is

Reply:
ty!